cedar-lean/Cedar/Thm/Authorization/Evaluator.lean

```diff
@@ -118,14 +117,14 @@ theorem ways_and_can_error {e₁ e₂ : Expr} {request : Request} {entities : En
           cases h_e₂ : (evaluate e₂ request entities) with
           | ok val =>
             cases val <;>
-            simp [h_e₂, evaluate, Result.as, Coe.coe, Value.asBool, Lean.Internal.coeM, pure, Except.pure] at h₁ <;>
+            simp [h_e₂, evaluate, Result.as, Value.asBool, pure, Except.pure] at h₁ <;>
             simp [h₁]
             case prim prim =>
               cases prim <;>
               simp [h_e₂] at h₁ <;>
               simp [h₁]
           | error e =>
-            simp [h_e₂, Lean.Internal.coeM] at h₁
+            simp only [h_e₂, ↓reduceIte, Except.bind_err, Except.error.injEq] at h₁
```

Comment:
Lean noob question: is the downward arrow some special syntax?

Reply:
Not sure myself.  `simp?` suggested this

Reply:
When I mouse over it tells me it tells Lean to apply this before entering the binders instead of after